Jetcaster/wear/src/main/java/com/example/jetcaster/MainActivity.kt

```diff
@@ -78,16 +94,15 @@ fun WearApp() {
                 )
             },
             libraryScreen = {
-//                val columnState = rememberColumnState()
-
-//                ScreenScaffold(scrollState = columnState) {
-//                        JetcasterBrowseScreen(
-//                            jetcasterBrowseScreenViewModel = JetcasterBrowseScreenViewModel(),
-//                            onPlaylistsClick = { navController.navigateToCollections() },
-//                            onSettingsClick = { navController.navigateToSettings() },
-//                            columnState = columnState,
-//                        )
-//                }
+                ScreenScaffold() {
```

Comment:
Should `ScreenScaffold` be an implementation detail within `HomeScreen`? This way, you can call `HomeScreen` as-is.

Reply:
done